docs/configuration/digid_oidc.rst

```diff
@@ -0,0 +1,195 @@
+.. _digid_oidc:
+
+
+========================
+DigiD OIDC configuration
+========================
+
+
+Settings Overview
+=================
+
+Required:
+"""""""""
+
+::
+
+    DIGID_OIDC_OIDC_RP_CLIENT_ID
+    DIGID_OIDC_OIDC_RP_CLIENT_SECRET
+    
+
+
+All settings:
+"""""""""""""
+
+::
+
+    DIGID_OIDC_ENABLED
+    DIGID_OIDC_ERROR_MESSAGE_MAPPING
+    DIGID_OIDC_IDENTIFIER_CLAIM_NAME
+    DIGID_OIDC_OIDC_EXEMPT_URLS
+    DIGID_OIDC_OIDC_KEYCLOAK_IDP_HINT
+    DIGID_OIDC_OIDC_NONCE_SIZE
+    DIGID_OIDC_OIDC_OP_AUTHORIZATION_ENDPOINT
+    DIGID_OIDC_OIDC_OP_DISCOVERY_ENDPOINT
+    DIGID_OIDC_OIDC_OP_JWKS_ENDPOINT
+    DIGID_OIDC_OIDC_OP_LOGOUT_ENDPOINT
+    DIGID_OIDC_OIDC_OP_TOKEN_ENDPOINT
+    DIGID_OIDC_OIDC_OP_USER_ENDPOINT
+    DIGID_OIDC_OIDC_RP_CLIENT_ID
+    DIGID_OIDC_OIDC_RP_CLIENT_SECRET
+    DIGID_OIDC_OIDC_RP_IDP_SIGN_KEY
+    DIGID_OIDC_OIDC_RP_SCOPES_LIST
+    DIGID_OIDC_OIDC_RP_SIGN_ALGO
+    DIGID_OIDC_OIDC_STATE_SIZE
+    DIGID_OIDC_OIDC_USE_NONCE
+    DIGID_OIDC_USERINFO_CLAIMS_SOURCE
+    
+
+
+Detailed Information
+====================
+
+::
+
+    Variable            DIGID_OIDC_OIDC_RP_CLIENT_ID
+    Setting             OpenID Connect client ID
+    Description         OpenID Connect client ID provided by the OIDC Provider
+    Model field type    CharField
```

Comment:
I know we need this to figure out the possible values, but we probably don't need to display it in the actual docs